keyboards/scorpius_m32/keymaps/default/config.h

```diff
@@ -0,0 +1,3 @@
+#pragma once
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15


ALso, this should probably be moved to the via folder, or set in the keyboard's config.h file

Reply:
> Also, this should probably be moved to the via folder, or set in the keyboard's config.h file

Reply:
I have finished the fix.